<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleWizardPage</class>
 <widget class="QWizardPage" name="ScheduleWizardPage">
  <attribute name="title">
    <string>Schedule</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="TextLabel22_2">
        <property name="text">
          <string>KMyMoney will create a schedule for this payment and reminds you whenever a payment must be made.&lt;p&gt;
If you selected to record all payments this date has already been supplied. If you selected to record only this years payments, then the &lt;b&gt;First payment due date&lt;/b&gt; is the date of the first payment made in this year.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer67_3">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>23</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QGridLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item row="1" column="1">
          <widget class="KMyMoneyAccountSelector" name="m_paymentAccountEdit"/>
        </item>
        <item row="0" column="1">
          <widget class="KMyMoneyDateEdit" name="m_nextDueDateEdit"/>
        </item>
        <item row="0" column="0">
          <widget class="QLabel" name="m_dueLabel">
            <property name="text">
              <string>First payment due on:</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item row="1" column="0">
          <widget class="QLabel" name="TextLabel2_5">
            <property name="text">
              <string>Make payment from/to:</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer68_2">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>24</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyDateEdit</class>
   <extends>KDateComboBox</extends>
   <header>kmymoneydateedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyCategory</class>
   <extends>QWidget</extends>
   <header>kmymoneycategory.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
